WMFData/Sources/WMFData/Models/Donor Experience/WMFDonateConfig.swift

```diff
@@ -13,4 +13,15 @@ public struct WMFDonateConfig: Codable {
     public func transactionFee(for currencyCode: String) -> Decimal? {
         return currencyTransactionFees[currencyCode] ?? currencyTransactionFees["default"]
     }
+
+    public func getMaxAmount(for currencyCode: String) -> Decimal {
+        var max = currencyMaximumDonation[currencyCode] ?? Decimal()
+
+        if max.isZero {
+            if let defaultMin = currencyMinimumDonation["USD"], let defaultMax = currencyMaximumDonation["USD"], let currencyMax = currencyMinimumDonation[currencyCode] {
```

Comment:
Minor - I think the final var here should be named `let currencyMin` instead of `let currencyMax`.

Reply:
I opened another PR to fix that https://github.com/wikimedia/wikipedia-ios/pull/4961